server/repository/src/migrations/v2_01_00/name_properties/name_property_data.rs

```diff
@@ -0,0 +1,28 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    // Create the name properties available for GAPS - UI to configure these may come at a later date
+    sql!(
+        connection,
+        r#"
+        INSERT INTO property (id, key, name, value_type, allowed_values) VALUES 
+          ('supply_level', 'supply_level', 'Supply Level', 'STRING', 'Primary,Sub-National,Lowest Distribution,Service Point'),
+          ('facility_type', 'facility_type', 'Facility Type', 'STRING', 'National Vaccine Store,Regional Vaccine Store,Referral Hospital,Municipal Warehouse,Maternal Clinic'),
+          ('ownership_type', 'ownership_type', 'Ownership Type', 'STRING', 'Government,NGO,Private,Faith-based'),
+          ('electricity_availability', 'electricity_availability', 'Electricity Availability', 'STRING', '> 16 hours,8-16 hours,< 8 hours,No availability'),
+          ('solar_availability', 'solar_availability', 'Solar Availability', 'STRING', '> 16 hours,8-16 hours,< 8 hours,No availability'),
+          ('gas_availability', 'gas_availability', 'Gas Availability', 'STRING', '> 16 hours,8-16 hours,< 8 hours,No availability'),
+          ('kerosene_availability', 'kerosene_availability', 'Kerosene Availability', 'STRING', '> 16 hours,8-16 hours,< 8 hours,No availability');
```

Comment:
Do you think we should make this a JSON Array?
I copied the existing pattern for asset properties, but when we're making something more generic I think we probably should make it more robust?

Reply:
Would want to re-factor asset_properties too.
Maybe we don't worry for now?